qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteredSnapshotTest.java

```diff
@@ -31,14 +31,15 @@
 import java.util.stream.IntStream;
 import org.assertj.core.api.AbstractAssert;
 import org.assertj.core.api.Assertions;
+import org.awaitility.Awaitility;
 import org.junit.After;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameter;
 import org.junit.runners.Parameterized.Parameters;
-import org.testcontainers.shaded.org.awaitility.Awaitility;
+import org.springframework.util.unit.DataSize;
```

Comment:
:x: :detective: is this supposed to be here?

Reply:
Ah thanks, that was a merge mistake. Should be fixed now